cloud/tasks/storage/storage_ydb_impl.go

```diff
@@ -686,14 +685,14 @@ func (s *storageYDB) createRegularTasks(
 	defer res.Close()
 
 	found := false
-	sch := schedule{}
+	schState := scheduleState{}
```

Comment:
Why not just create a ScheduleState variable, schState is understandable given the context, yet I think that we should measure up to higher expectations regarding the readability of the code.